kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/clients/TestClient.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.clients;
+
+import java.time.Duration;
+
+import io.fabric8.kubernetes.api.model.batch.v1.Job;
+
+import io.kroxylicious.systemtests.Constants;
+import io.kroxylicious.systemtests.templates.testclients.TestClientsJobTemplates;
+import io.kroxylicious.systemtests.utils.KafkaUtils;
+
+import static io.kroxylicious.systemtests.k8s.KubeClusterResource.kubeClient;
+
+/**
+ * The type Test client (java client based CLI).
+ */
+public class TestClient implements KafkaClient {
```

Comment:
I was thinking about this name for a while as well and I didn't find any proper name. I'd go with `CliClient` because `ApacheKafkaClient` could be confusing as well as the interface is called `KafkaClient`.